plugins/cloudtrail/CHANGELOG.md

```diff
@@ -1,10 +1,18 @@
 # Changelog
 
+## v0.12.2
+
+* [`63b7093`](https://github.com/falcosecurity/plugins/commit/63b7093) chore(plugin/cloudtrail): bump cloudtrail version to 0.10.0
```

Comment:
This is here because I'm dumb and I wrote `cloudtrail` instead of `k8saudit`. Don't worry about it.